x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.container.js

```diff
@@ -9,36 +9,35 @@ import { connect } from 'react-redux';
 import { SECTIONS } from '../../../constants';
 import {
   getListAutoFollowPatterns,
+  getSelectedAutoFollowPatternId,
   getApiStatus,
   getApiError,
   isApiAuthorized,
-  isAutoFollowPatternDetailPanelOpen as isDetailPanelOpen,
+  isAutoFollowPatternDetailPanelOpened as isDetailPanelOpened,
 } from '../../../store/selectors';
 import {
   loadAutoFollowPatterns,
-  openAutoFollowPatternDetailPanel as openDetailPanel,
-  closeAutoFollowPatternDetailPanel as closeDetailPanel,
+  selectAutoFollowPattern,
+  toggleAutoFollowPatternDetailPanel,
 } from '../../../store/actions';
 import { AutoFollowPatternList as AutoFollowPatternListView } from './auto_follow_pattern_list';
 
 const scope = SECTIONS.AUTO_FOLLOW_PATTERN;
 
 const mapStateToProps = (state) => ({
   autoFollowPatterns: getListAutoFollowPatterns(state),
+  autoFollowPatternId: getSelectedAutoFollowPatternId(state),
   apiStatus: getApiStatus(scope)(state),
   apiError: getApiError(scope)(state),
   isAuthorized: isApiAuthorized(scope)(state),
-  isDetailPanelOpen: isDetailPanelOpen(state),
+  isDetailPanelOpened: isDetailPanelOpened(state),
```

Comment:
Minor nit, but I think we should use the present tense in all booleans for consistency, unless the tense has some clear significance.

Reply:
Ok! Was not sure if it was a mistake. So you prefer `isClose` to `isClosed` for example?

Reply:
English is such a terrible language... "closed" is both the present and past tense, so complementary states would be `isClosed` and `isOpen`.